compare.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-
+rm -f old.json te.json
```

Comment:
huh, was it appending not overwriting?

Reply:
This script is running without `-e` so failures are ignored (sometimes tests fail, so this is the right behavior).

I wanted to be paranoid and doubly make sure there wasn't some failure that causes us to pick up old data in a later step.

(This is more of an "avoid nasty gotchas" change than a bugfix.)